cmd/config/config.go

```diff
@@ -1,21 +1,54 @@
 package config
 
 import (
-	"github.com/pingidentity/pingctl/cmd/config/profile"
+	"os"
+
+	"github.com/pingidentity/pingctl/cmd/common"
+	config_internal "github.com/pingidentity/pingctl/internal/commands/config"
+	"github.com/pingidentity/pingctl/internal/configuration"
+	"github.com/pingidentity/pingctl/internal/logger"
 	"github.com/spf13/cobra"
 )
 
 func NewConfigCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Long:  `Command to get, set, and unset pingctl configuration settings.`,
-		Short: "Command to get, set, and unset pingctl configuration settings.",
-		Use:   "config",
+		Args:                  common.ExactArgs(0),
+		DisableFlagsInUseLine: true, // We write our own flags in @Use attribute
+		Example: `pingctl config
+pingctl config --profile myprofile
+pingctl config --name myprofile --description "My Profile"`,
+		Long:  `Update a configuration profile's name and description. See subcommands for more profile configuration management.`,
+		RunE:  configRunE,
+		Short: "Update a configuration profile's name and description. See subcommands for more profile configuration management.",
```

Comment:
Yes, the config command updates a profile, in addition to being the parent of additional subcommands. 

What do you mean by missing a final word?

Reply:
Is "Update a configuration profile's name and description." supposed to be a description of the entirety of what the 'config' command does? That was my assumption and why I commented on it, because it doesn't seem to sufficiently describe what this command is for.

I guess "See subcommands for more profile configuration management." sounds awkward to me. Like it should be "See subcommands for more profile configuration management options." for example. Just my opinion on that one.

Reply:
Makes sense! I'll update the verbiage. Thanks!